src/main/java/subway/exception/InvalidInputException.java

```diff
@@ -0,0 +1,15 @@
+package subway.exception;
+
+import lombok.Getter;
+
+@Getter
+public class InvalidInputException extends RuntimeException {
+    private final String code;
+    private final String message;
+
+    public InvalidInputException(String message) {
+        super();
+        code = "INVALID_INPUT";
+        this.message = message;
+    }
+}
```

Comment:
일반적으로 예외 클래스에서 필드를 선언하지 않고 상속받은 RuntimeException의 생성자를 호출하여 예외 객체를 생성할 수 있습니다.

아래와 같이 RuntimeException을 상속받은 예외 클래스를 만들 수 있습니다.
```suggestion
public class InvalidInputException extends RuntimeException {
    public InvalidInputException(String message) {
        super(message);
    }
}
```
여기서 RuntimeException의 생성자를 호출하여 예외 객체를 생성하면, 
부모 클래스에서 이미 예외 메시지를 처리하는 로직이 구현되어 있기 때문에 별도의 필드를 선언할 필요가 없는데요.

code와 message 필드를 추가해주신 이유가 더 많은 정보를 제공하기 위함인지 궁금합니다 🧐

Reply:
아닙니다! 별도의 필드를 선언할 필요가 없는지 몰랐습니다 ㅎㅎ...